components/gantry/multiaxis/multiaxis.go

```diff
@@ -15,22 +15,24 @@ import (
 	"go.viam.com/rdk/resource"
 )
 
-var model = resource.DefaultModelFamily.WithModel("multiaxis")
+var model = resource.DefaultModelFamily.WithModel("multi-axis")
 
 // Config is used for converting multiAxis config attributes.
 type Config struct {
-	SubAxes []string `json:"subaxes_list"`
+	SubAxes            []string `json:"subaxes_list"`
+	MoveSimultaneously *bool    `json:"move_simultaneously,omitempty"`
 }
 
 type multiAxis struct {
 	resource.Named
 	resource.AlwaysRebuild
-	subAxes   []gantry.Gantry
-	lengthsMm []float64
-	logger    golog.Logger
-	model     referenceframe.Model
-	opMgr     operation.SingleOperationManager
-	workers   sync.WaitGroup
+	subAxes            []gantry.Gantry
+	lengthsMm          []float64
+	logger             golog.Logger
+	moveSimultaneously bool
+	model              referenceframe.Model
+	opMgr              operation.SingleOperationManager
+	workers            sync.WaitGroup
```

Comment:
Seems like multiaxis is missing Reconfigure function.

Reply:
that will be addressed later when I work on multi axis functionality! this is just about config changes